doc/10_GraphQL/04_Query/11_Query_Samples/12_Sample_Asset_Metadata.md

```diff
@@ -80,4 +80,99 @@ Get the custom asset metadata for language `de`
 }
 ```
 
+## Get Asset Embedded Meta Info
 
+![Metadata](../../../img/graphql/asset_embeddedMetaInfo.png)
+
+### Request
+
+Get the embedded meta info asset metadata for an asset
+
+```graphql
+{
+  getAsset(id: 295) {
+    embeddedMetaInfo {
+      name
+      value
+    }
+  }
+}
+```
+
+### Response
+
+```graphql
```

Comment:
I used "graphql" here to be in sync with the rest of the document, but actually the response is `json` and you might consider changing that for a proper highlighting :)

Reply:
Good point, thanks, it is fine for now, but I created a follow up ticket
https://github.com/pimcore/data-hub/issues/691